db/offchaindb/db.go

```diff
@@ -0,0 +1,483 @@
+// Copyright (c) 2019, The Decred developers
+// See LICENSE for details.
+
+package offchaindb
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/asdine/storm"
+)
+
+const (
+	// vettedProposalsRoute is the URL path that returns the public/vetted
+	// proposals.
+	vettedProposalsRoute = "/proposals/vetted"
```

Comment:
Please use the constants in https://github.com/decred/politeia/blob/master/politeiawww/api/v1/v1.go

```go
import piapi "github.com/decred/politeia/blob/master/politeiawww/api/v1"
```

Then you just use `piapi.RouteAllVetted` and `piapi.RouteAllVoteStatus` (or any other routes named there).